README.Rmd

```diff
@@ -26,75 +26,91 @@ knitr::opts_chunk$set(
 [![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/pak/master.svg)](https://codecov.io/github/r-lib/pak?branch=master)
 <!-- badges: end -->
 
-pak installs R packages from various sources.
+pak installs R packages from various sources. It is a replacement for
+`install.packages()` and `devtools::install_github()`.
+pak is fast, safe and convenient.
+
+<p align="center">
+	<img width="1000" src="https://cdn.jsdelivr.net/gh/r-lib/pak@readme/tools/images/install-2.svg">
+</p>
+
 
 ## Installation
 
 Install the package from CRAN:
 
-``` r
+```r
 install.packages("pak")
 pak::pak_create_private_lib()
 ```
 
 The second line creates pak's own package library, to avoid interference
-between pak's dependencies and the user's regular packages. Run this
-command any time to update pak's package library.
+between pak's dependencies and the user's regular packages.
 
 ## Usage
 
-Simply call `pkg_install` to install packages:
+Simply call `pkg_install()` to install CRAN or BioConductor packages:
 
 ```r
-pak::pkg_install("dplyr", lib = "/tmp/lib")
+pak::pkg_install("usethis")
 ```
 
-All dependencies will be installed as well, to the same library.
-
-## Goals
+To install GitHub packages, use the `user/repo` syntax:
```

Comment:
This is where the traditional "what about GitLab/Bitbucket" questions will come. I'd mention the support for that (or lack thereof) once, somewhere.

Reply:
They are coming soon, hopefully, so maybe not worth making a big deal?

Reply:
Sure. I just want to be on record as predicting the question 😄

Reply:
Maybe I'll just add them quickly, they are easy, anyway....

Reply:
I addd a Roadmap section where I mention other remote types.